### ISSUE TITLE
Spike 2 - What are the community's needs for parent/child relationships?

### ISSUE THREAD

**Issue by ShanaLMoore** [notch8/utk-hyku#156]
ref #83 

Do we have a list of features implemented for LV? 

This ticket is about comparing the needs of our three clients: LV, IU, and UTK. 

IU SOW: https://docs.google.com/document/d/1JeHrZGkFtGYPbekWkakVqaBW7ZHhBcZlNAVxDtjH190/edit

UTK's [Compound Objects.](https://utk-future-dc-worktypes.readthedocs.io/en/latest/contents/7_compound_objects.html) 
- [SOW](https://docs.google.com/document/d/1tym1MPANEZgTZYaEEyRw0aGVvC7RB_WJGMA1VgcrBcQ/edit)

**Comment by ShanaLMoore** [comment]
LOUISVILLE #155 

- only return parents in catalog search by default
  - children can be returned when clicking on facetable links from within the parent's UV
- parent UV displays all of its children, and its childrens' metadata in the side pane
- the side pane metadata changes when you click on different children, within the parent's UV
- A user is able to search for child OCR within the parent UV. 
- OCR matches highlight the words in the UV. It also highlights the border of the child it belongs to
- A user can also search for OCR within the child's UV and find highlighted matches
- A user is able to search for child metadata within the parent UV and have the results' border highlighted
- When the parent has no filesets, its thumbnail should be the first child's first fileset
- A non admin user can download low resolution filesets only
- pdf is rendered in the uv

**Comment by ShanaLMoore** [comment]
INDIANA UNIVERSITY
- ~~they want similar functionality but for their collections instead of works~~
- desire is they can apply this to any worktype, but esp for Newspapers
-  allow children works to show in the viewer of a parent
-  allow child metadata to be available in the parent UV when navigating between pages
- UV objects rendered ordered by identifier

- When publicly viewed, displays child/member works in a single viewer
- All child work thumbnails display in the items pane
- Individual child/member work metadata can be viewed in the “more information” tab, which currently only shows a subset of metadata. The initial requirement is for all publicly viewable (exposed) metadata to appear in the viewer. If display becomes too crowded, Indiana can add a tag to metadata they would like to appear in the viewer.
- Parent-level metadata can also be viewed in viewer, similar to individual work
- Navigation tool to move to next child/member work (if possible)
- Search in the viewer searches all collection content, not just the current loaded page or child/member item (if possible)

Modeled on successful work with University of Louisville, SoftServ will use parent/child hierarchy to achieve the goal specific to collection and item. The Newspaper Gem utilizes a model where Newspaper is the parent, and Page is the child.  The gem can be utilized for other works that follow the same model, and will meet the requirements for medieval manuscripts and other works. It will allow:
- All pages when viewing the work (e.g., newspaper, manuscript) to load in the UV
- Search across all pages contained in the work

- If it doesn’t pose any significant technical challenges, SoftServ will target IIIF Presentation 3

**Comment by ShanaLMoore** [comment]
UTK [SOW](https://docs.google.com/document/d/1tym1MPANEZgTZYaEEyRw0aGVvC7RB_WJGMA1VgcrBcQ/edit)

[COMPOUND OBJECTS](https://utk-future-dc-worktypes.readthedocs.io/en/latest/contents/7_compound_objects.html) = works that are made up of 2 or more works

- 
- All compound objects’ individual works’ filesets should be available for download.
- Book works within a compound object work should have OCR and HOCR.
- All audio and video works within a compound object work may have 0-n closed-captioning / subtitle files that should be associated with it in the viewer.
- All filesets may have restrictions that prohibit view / access.
- Compound object works should be displayed in a IIIF viewer such as Universal Viewer, Clover IIIF, or Mirador with an individuals behavior at the manifest level. 
- The IIIF viewer should provide pan and zoom from the embedded OpenSeaDragon viewer on image subworks and closed captioning on audio / video sub works. Our current IIIF configuration along with “viewing experience” for compound objects is described in our [Compound Objects recipe document](https://utk-iiif-cookbook.readthedocs.io/en/latest/contents/compound_objects.html).
- The pcdmuse:IntermediateFile should always be represented in the viewer with any corresponding files (Transcripts).
- Location based viewing
- rights viewing
  - While we store our rights as uris (see metadata application profile), we’d like these statements to be actionable and render the appropriate badge and usage information to the user in both the IIIF viewer (when metadata is on) and in our metadata display below.
- File Viewing 
  - Users should be able to easily download the primary filesets.
  - If a fileset is only a pcdm:PreservationFile it should never be publicly visible.
-

**Comment by ShanaLMoore** [comment]
**TLDR:** It looks like both UTK and IU will benefit from the work done in LV. Additional customization may need to happen for each app. 


- [ ] Parent UV should display child works and its metadata
- [ ] A user should be able to search and find its children's OCR and metadata from the parent UV.
- [ ] pull in newspaper work/nnp?
- [ ] no download restrictions
- [ ] A catalog search should return both parents and children?

**Comment by ShanaLMoore** [comment]
Rob suggested that we turn our Spike 3 implementation meeting to a 2 hr pairing session to tease out the bits of the newspaper gem we'd want to keep. This meeting has been scheduled for Wednesday 9 to 11 AM PST.

**Comment by ShanaLMoore** [comment]
Results will be completed as part of #157
